File: ESTUDO DE LINGUAGENS/PYTHON/Questões Uri/2455.py
a,b,c,d=map(int, input().split(" "))

soma=((a*b)-(c*d))

if (soma==0):
  print("0")
elif ((a*b)>(c*d)):
  print("-1")
elif((a*b)<(c*d)):
  print("1")

File: ESTUDO DE LINGUAGENS/JAVASCRIPT/Questões uri/2455.js
const pegarValoress = (line) => line.split(" ").map(a => +a)
const [a,b,c,d] = pegarValoress(lines.shift())

let soma=((a*b)-(c*d))

if (soma==0){
  console.log("0")
}
else if ((a*b)>(c*d)){
  console.log("-1")
}
else if((a*b)<(c*d)){
  console.log("1")
}